.github/workflows/maven.yml

```diff
@@ -0,0 +1,13 @@
+name: Verify
+
+on:
+  push:
```

Comment:
only on `main` here.

Reply:
so it means we are not be able to branch before creating a PR